contracts/connectors/loantoken/modules/LoanTokenLogicLM.sol

```diff
@@ -59,26 +47,27 @@ contract LoanTokenLogicLM is LoanTokenLogicStandard {
 		res[24] = this.checkPriceDivergence.selector;
 		res[25] = this.checkPause.selector;
 		res[26] = this.setLiquidityMiningAddress.selector;
+		res[27] = this._supplyInterestRate.selector;
```

Comment:
consider renaming this function if possible, `_` is not supposed to be part of the function name at all (according to Solidity style guide), especially for those available externally.